_posts/2020-06-14-dynamic-partition-pruning.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+title:  "Dynamic partition pruning"
+author: Raunaq Morarka, Qubole and Karol Sobczak, Starburst Data
+excerpt_separator: <!--more-->
+---
+
+Improving dynamic filtering to speed up inner-joins on partitioned tables.
```

Comment:
Make this section a bit longer so more is displayed on the blog index page and word it so the reader wants to learn more and go to the full post.

Reply:
Moved introduction here